src/pages/FAQ.jsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
+import Box from "@material-ui/core/Box";
 import AccordionList from "../components/AccordionList";
 
 const FAQ = () => (
-  <div className="FAQ" data-test="FAQ-page">
-    <h1>FAQ</h1>
-    <AccordionList />
-  </div>
+  <Box mt="40" mx="auto ">
+    <div className="FAQ" data-test="FAQ-page">
+      <h1>FAQ</h1>
+      <AccordionList />
```

Comment:
🔧 add tests for these items

Reply:
Addressed: e67701b768282739f6b632184983497816fa6ab0